.ci-tools/vizabi.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -e
+
+FORBIDDEN_BRANCHES="master development"
+CURRENT_BRANCH="$(git branch -q --no-color --contains | awk {'print $2'})"
```

Comment:
If i'll use 'TRAVIS_BRANCH' env variable, then this script will not work on local workstations.
If it is OK - i'll make changes.